docs/source/quickstart/index.rst

```diff
@@ -0,0 +1,717 @@
+            "subscriptions_url": "https://api.github.com/users/dwalleck/subscriptions",
+            "avatar_url": "https://avatars2.githubusercontent.com/u/843116?v=3",
+            "repos_url": "https://api.github.com/users/dwalleck/repos",
+            "received_events_url": "https://api.github.com/users/dwalleck/received_events",
+            "gravatar_id": "",
+            "starred_url": "https://api.github.com/users/dwalleck/starred{/owner}{/repo}",
+            "site_admin": false,
+            "login": "dwalleck",
+            "type": "User",
+            "id": 843116,
+            "followers_url": "https://api.github.com/users/dwalleck/followers"
+        },
+        "milestone": null,
+        "closed_by": {
+            "following_url": "https://api.github.com/users/jidar/following{/other_user}",
+            "events_url": "https://api.github.com/users/jidar/events{/privacy}",
+            "organizations_url": "https://api.github.com/users/jidar/orgs",
+            "url": "https://api.github.com/users/jidar",
+            "gists_url": "https://api.github.com/users/jidar/gists{/gist_id}",
+            "html_url": "https://github.com/jidar",
+            "subscriptions_url": "https://api.github.com/users/jidar/subscriptions",
+            "avatar_url": "https://avatars2.githubusercontent.com/u/1134139?v=3",
+            "repos_url": "https://api.github.com/users/jidar/repos",
+            "received_events_url": "https://api.github.com/users/jidar/received_events",
+            "gravatar_id": "",
+            "starred_url": "https://api.github.com/users/jidar/starred{/owner}{/repo}",
+            "site_admin": false,
+            "login": "jidar",
+            "type": "User",
+            "id": 1134139,
+            "followers_url": "https://api.github.com/users/jidar/followers"
+        },
+        "locked": false,
+        "url": "https://api.github.com/repos/CafeHub/opencafe/issues/42",
+        "created_at": "2017-02-27T18:34:21Z",
+        "assignees": []
+    }
+
+The BaseHTTPClient returns the same response that ``requests`` would, so we
+can treat the response similarly to view its content. At this point, it
+doesn't look like the OpenCafe HTTP plugin is adding any more value than
+``requests`` would. Let's see what we can do about that. First, let's setup
+logging and see what happens.
+
+.. code-block:: python
+
+    import json
+    import logging
+    import os
+    import sys
+
+    from cafe.engine.http.client import BaseHTTPClient
+    from cafe.common.reporting import cclogging
+
+
+    os.environ['CAFE_ENGINE_CONFIG_FILE_PATH']='.'
+    cclogging.init_root_log_handler()
+    root_log = logging.getLogger()
+    root_log.addHandler(logging.StreamHandler(stream=sys.stderr))
+    root_log.setLevel(logging.DEBUG)
+
+    client = BaseHTTPClient()
+    response = client.get('https://api.github.com/repos/cafehub/opencafe/issues/42')
+
+**Review note: Jose, correct me if I'm not clear on the cclogging bits**
```

Comment:
Yup.  If defines methods for helping generate loggers with a uniform format and interface across the project.